BREAKING.md

```diff
@@ -134,8 +134,12 @@ The following members of the `@fluidframework/driver-utils` package have been de
 The following members of the `@fluidframework/aqueduct` package have been deprecated and will be removed in an upcoming release:
 
 -   `waitForAttach()`
+
     -   Prefer not to inspect and react to the attach state unless necessary. If needed, instead inspect the IFluidDataStoreRuntime's attachState property, and await the "attached" event if not attached.
 
+-   `BaseContainerService`, `ContainerServiceRegistryEntries`, `generateContainerServicesRequestHandler()` and `serviceRoutePathRoot`
```

Comment:
Ultimately it doesn't really matter, so ignore if this doesn't work.  But I mean get rid of both newlines not just one.

```suggestion
-   `waitForAttach()`
    -   Prefer not to inspect and react to the attach state unless necessary. If needed, instead inspect the IFluidDataStoreRuntime's attachState property, and await the "attached" event if not attached.
-   `BaseContainerService`, `ContainerServiceRegistryEntries`, `generateContainerServicesRequestHandler()` and `serviceRoutePathRoot`
```